README.md

```diff
@@ -34,15 +36,6 @@ Don't forget to manually install peer dependencies (`react`) if you use npm@3.
 (Module exposed as `CopyToClipboard`)
 ```
 
-
```

Comment:
I'd really prefer to keep these

You can add jinno demo with it's own header.

Reply:
Done